src/core/raster/qgsrasterlayertemporalproperties.cpp

```diff
@@ -133,7 +133,6 @@ void QgsRasterLayerTemporalProperties::setDefaultsFromDataProviderTemporalCapabi
 {
   if ( const QgsRasterDataProviderTemporalCapabilities *rasterCaps = dynamic_cast< const QgsRasterDataProviderTemporalCapabilities *>( capabilities ) )
   {
-    setIsActive( rasterCaps->hasTemporalCapabilities() );
```

Comment:
Why this change?

Reply:
When updating the WMS layer uri, the setDataSourceUri function is called to save changes, if the temporal properties where set to be inactive  by user, the above line will cause to reset them to be active.